collection.js

```diff
@@ -0,0 +1,49 @@
+const LOCAL_STORAGE_KEY = 'books';
+const list = document.getElementById('books');
+
+const getFromLocalStorage = () => {
+  const cachedBooks = window.localStorage.getItem(LOCAL_STORAGE_KEY);
+  return cachedBooks ? JSON.parse(cachedBooks) : [];
+};
+
+let booksArr = getFromLocalStorage();
+
+const saveToLocalStorage = () => {
+  window.localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(booksArr));
+};
+
+const deleteBook = (i) => {
```

Comment:
- [ ] **[OPTIONAL]** I would suggest that you use meaningful variable names, in this case, `i` doesn't provide a lot of information about what you are expecting as a parameter. 🤓 🔧